src/components/modals/RedeemModal.tsx

```diff
@@ -83,27 +83,16 @@ export default function RedeemModal({
     : rewardAmount
 
   function redeem() {
-    if (!transactor || !contracts || !rewardAmount || !terminal) return
+    if (!minAmount) return
 
     setLoading(true)
 
-    const redeemWad = parseWad(redeemAmount)
-
-    if (!redeemWad || !projectId) return
-
-    transactor(
-      terminal.version === '1.1'
-        ? contracts.TerminalV1_1
-        : contracts.TerminalV1,
-      'redeem',
-      [
-        userAddress,
-        projectId.toHexString(),
-        redeemWad.toHexString(),
+    redeemTokensTx(
+      {
+        redeemAmount: parseWad(redeemAmount),
         minAmount,
-        userAddress,
-        false, // TODO preferconverted
-      ],
+        preferConverted: false, // TODO support in UI
```

Comment:
**question**: what does this do of interest?

Reply:
converted is another word for "claimed/erc20". `preferConverted = true` means that if a user has both erc20 and unclaimed tokens, and redeems less than their entire token balance, the erc20 tokens will be redeemed first.

Reply:
oh nice I see, thanks for the explanation. What does the behavior when `false`? Is it just the opposite (unclaimed is first-out)?

Reply:
Answered my own question: **yes** https://docs.juicebox.money/protocol/specifications/contracts/jbtokenstore/write/burnfrom